src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public class Excavator extends Machine{
+
+
+    @Override
+    public void doWork() {
+        System.out.println("Excavator started working");
```

Comment:
The message in 'doWork' method is not informative enough. According to the checklist, it should indicate what type of Machine is working. Consider specifying the action that Excavator is performing, not just that it started working.